test/integration/serviceaccount/external_jwt_signer_test.go

```diff
@@ -280,9 +274,6 @@ func waitForDataTimestamp(t *testing.T, client kubernetes.Interface, minimumData
 }
 
 func TestDelayedStartForSigner(t *testing.T) {
-	// Enable feature gate for external JWT signer.
-	featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.ExternalServiceAccountTokenSigner, true)
```

Comment:
revert this change, continuing to set the feature gate to true explicitly is fine (and keeps this test passing if we have to toggle the gate back off by default for some reason)